packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -172,13 +171,15 @@ export function TableNG(props: TableNGProps) {
     () => (hasNestedFrames ? width - COLUMN.EXPANDER_WIDTH : width) - scrollbarWidth,
     [width, hasNestedFrames, scrollbarWidth]
   );
-  const typographyCtx = useTypographyCtx();
+  const typographyCtx = useMemo(
+    () => createTypographyContext(theme.typography.fontSize, theme.typography.fontFamily),
+    [theme]
+  );
```

Comment:
I lifted this up in anticipation that we may actually want to have two different typography contexts pretty soon. it came to our attention that there's movement around having monospace numbers, and so we'd want to have one for the standard font family and one for the monospace font family and change which one we use for row height calculations depending on the field type.